mmdeploy/apis/onnx/optimizer.py

```diff
@@ -5,8 +5,9 @@
 
 
 @FUNCTION_REWRITER.register_rewriter('torch.onnx.utils._model_to_graph')
-def model_to_graph__custom_optimizer(ctx, *args, **kwargs):
+def model_to_graph__custom_optimizer(*args, **kwargs):
     """Rewriter of _model_to_graph, add custom passes."""
+    ctx = FUNCTION_REWRITER.get_context()
```

Comment:
could update the `rewriting demo code` in docs as well,
for instance , this doc:
https://github.com/open-mmlab/mmdeploy/edit/dev-1.x/docs/en/07-developer-guide/support_new_model.md